plugins.json

```diff
@@ -1364,5 +1364,17 @@
         "requires": ">=22.11.0-edge"
       }
     ]
+  },
+  {
+    "id": "nf-cws", 
+    "releases": [
+      {
+        "version": "1.0.0",
+        "date": "2023-03-14T15:19:26.470272+01:00",
+        "url": "https://github.com/CommonWorkflowScheduler/nf-cws/releases/download/1.0.0/nf-cws-1.0.0.zip",
+        "requires": ">23.02.01-edge",
```

Comment:
Is not the plugin depending on the changes not yet released?

Reply:
I'm going to make a release early next week. What about waiting for it to merge this PR?

Reply:
Okay, let's wait for this.